src/Kernel/Class.class.st

```diff
@@ -1338,6 +1330,14 @@ Class >> usesPoolVarNamed: aString [
 	^self allSharedPools anySatisfy: [:each | each usesClassVarNamed: aString]
 ]
 
+{ #category : #compiling }
+Class >> variable [
```

Comment:
I think #variable is too simplistic name for this purpose. It's quite confusing due to existing instanceVariables, classVariables and other variants of actual variables defined by class.
Previous #binding was also not clear name but at least it was not related to actual variables of the class.

We can have something like:
```Smalltalk
aClass environmentVariable.
```
With the meaning of a variable containing the class in its environment.

Or maybe better:
```Smalltalk
aClass referenceVariable
```
With the meaning of a variable to reference the class. I think I would prefer this variant